src/server/transports/jsonrpc_transport_handler.ts

```diff
@@ -25,7 +25,7 @@ export class JsonRpcTransportHandler {
    * For non-streaming methods, it returns a Promise of a single JSONRPCMessage (Result or ErrorResponse).
    */
   public async handle(
-    requestBody: any,
+    requestBody: string | A2ARequest,
```

Comment:
Hmm, this is going to be a breaking change, this type is exported from `index.ts`. Let's suppress it here for now and add a TODO linking with #179 to fix later.